src/setup.ts

```diff
@@ -1,27 +1,37 @@
+import {getOctokit} from "@actions/github";
 import * as tc from "@actions/tool-cache";
 import * as core from "@actions/core";
 import fetch from "node-fetch"
 
-export async function getLatestVersion() {
+type Release = {
+  tag_name: string;
+};
+
+export async function getLatestVersion(token: string) {
   const url = "https://api.github.com/repos/loilo-inc/canarycage/releases";
-  type Release = {
-    tag_name: string;
-  };
-  const res = await fetch(url)
-  if (res.status === 200) {
-    const list: Release[] = await res.json()
+  if (token == "") {
```

Comment:
I think it's better to require tokens from next release.

Reply:
@keroxp Required github-token.